CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
-# 12.1.0
+# 12.1.1
```

Comment:
This should be `#HEAD` for this commit just to keep the history clean, and we'll version it in the other PR for publishing.

Reply:
Updating.